mu/harness/pm.py

```diff
@@ -123,6 +147,31 @@ def log_all(self):
                 continue
         self.logger.unsubscribe(sub)
 
+    def sim_list(self):
+        ret = []
+        for fd in self.fd_to_sim:
+            ret.append({
+                'fd': fd,
+                'sim': self.fd_to_sim[fd].__class__.__name__,
+            })
+        return ret
+
+    def sim_cat(self):
+        sim_files = os.listdir(os.path.join(REPO_DIR, 'mu/sims'))
+        catalog = {}
+        for sim_file in sim_files:
+            if not sim_file.endswith('.py'):
+                continue
+            sim_name = sim_file[:len(sim_file)-3]
+            mod_name = 'mu.sims.{}'.format(sim_name)
+            importlib.import_module(mod_name)
+            sim_classes = inspect.getmembers(sys.modules[mod_name], inspect.isclass)
+            for sim_class in sim_classes:
```

Comment:
Could make the loop a bit clearer by iterating over the pairs directly: `for sim_name, sim_class in sim_classes:`.